README.md

```diff
@@ -12,3 +12,10 @@ To get the latest development version of the snap, build from the source code an
 make build
 sudo snap install --dangerous prometheus-juju-exporter.snap
 ```
+
+## Configuration
+By default, prometheus-juju-exporter uses the configuration specified in [config_default.yaml](prometheus-juju-exporter/config_default.yaml). It's likely not fitting your need because the values provided are generic and for demonstration purposes only. To configure the snap for your own environment, create a `config.yaml` file in `$SNAP_DATA` (see [Snap Environment Variables](https://snapcraft.io/docs/environment-variables)) directory with the desired entries.
+
+The snap requires juju user's credentials to connect to a controller and all of its models. The credentials are set as `juju.username` and `juju.password` config values. Normally, **`superuser`** privilege is expected for such a user for full access to the controller. However, in case it is not possible, the minimum privilege requirements are:
+1. `login` access to controller, and
+2. `admin` access to all *active* models (including the `controller` model)
```

Comment:
```suggestion
1. `login` and `admin` access to `controller` model
2. `admin` access to any model that's expected to be monitored by this exporter

```
Just a small suggestion in wording.

Reply:
Thanks, I've applied your suggestion with some changes. For requirement 1, there are actually two different permissions needed. A user need to have `login` access to the controller instance, and `admin` access to models (including the model named `controller`).

Reply:
Oh, I missed that  distinction. thanks. LGTM.